third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -598,10 +571,22 @@ function Target_getCurrentNetworkStreamData(params) {
 }
 
 function Target_topFrameLocation() {
-  const { location } = sendMessage("Debugger.getTopFrameLocation");
-  if (!location) {
-    return {};
+  try {
+    const { location } = sendMessage("Debugger.getTopFrameLocation");
+    if (!location) {
+      return {};
+    }
+  } catch (e) {
+    if (e instanceof CDPMessageError) {
+      // No available context group; this can happen, so just return nothing.
+      if (e.code == CDPError_MissingContext) {
+        warning(`JS Target_topFrameLocation has no context.`);
```

Comment:
When we add these types of warnings, we should make sure that someone is actually tracking and triaging them.
We can add this for now, but ideally, we can discuss this aspect of our work tmrw or on Monday, since usually only I am doing it, and I'd prefer it, if we all participated in the error tracking + triaging process 👀

Reply:
Yup--but in this case, this isn't (necessarily?) an error to be tracked--this might all be fine and expected behaviour.

Reply:
We definitely want to tag them so we can easily group them and search for them.